### PR TITLE
Don't clone tahoe

### PR DESCRIPTION
**Pull request by tomprince**
Based on #403.

### CODE REVIEW

**Comment by crwood** [comment]
Thanks for this. Would you mind rebasing onto `master` (or merging `master` into your branch)? It should resolve the GitHub Action failure.

**Comment by tomprince** [comment]
Done.

File: misc/tahoe.spec [hunk 1]
```diff
@@ -16,34 +23,45 @@ options = [
...
+    """
+    This is a version of :py:`PyInstaller.utils.hooks.collect_dynamic_libs` that
+    will include linux shared libraries without a `lib` prefix.
+
+    It also only handles dymaic libraries in the root of the package.
+    """
+    base_path, pkg_path = get_package_paths(package)
+    pkg_rel_path = remove_prefix(pkg_path, base_path)
+    dylibs = []
+    for lib_ext in ["*.so", "*.dll", "*.dylib"]:
```
Comment (crwood) [inline misc/tahoe.spec:38]: As per the last line you removed in `make.bat`, this list should probably contain `*.pyd` also.

**Review by crwood — CHANGES_REQUESTED** [review]
This looks great; thank you!

Just one small suggested change below and I'm happy to merge.

File: misc/tahoe.spec (hunk 1 above)
Comment (tomprince) [reply misc/tahoe.spec]: Done.

Comment (crwood) [reply misc/tahoe.spec]: Thanks!

**Pull request merged**